internal/pkg/cli/storage_init.go

```diff
@@ -292,6 +476,11 @@ func logicalIDSafe(input string) string {
 	return nonAlphaNum.ReplaceAllString(input, "")
 }
 
+func (o *initStorageOpts) generateDynamoDBConfig() (*dynamoDBConfig, error) {
+	// TODO fill in as needed
+	return nil, nil
+}
+
```

Comment:
One thing that'd be neat for customers is if the flags appear grouped when they run `storage init --help`.
We can do something similar to "svc init":
https://github.com/aws/amazon-ecs-cli-v2/blob/098e612946d29475a8256556ee064c451e68da98/internal/pkg/cli/svc_init.go#L427-L437